custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -138,7 +138,8 @@ class ScenarioSimilarities(
             searchJclProbability = 0.05f,
             globalSearchProbability = 0.50f,
             customizeColumnsProbability = 0.05f,
-            switchBetweenIssuesProbability = 0.15f
+            switchBetweenIssuesProbability = 0.15f,
+            subscribeToFilterProbability = 0f
```

Comment:
```suggestion
            subscribeToFilterProbability = 0.00f
```
and comment on how much should it be if we can mutate the data